blocks/library/code-mirror/index.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * External dependencies
+ */
+import CodeMirror from 'react-codemirror';
+require( 'codemirror/mode/css/css' );
+require( 'codemirror/mode/diff/diff' );
+require( 'codemirror/mode/elm/elm' );
+require( 'codemirror/mode/javascript/javascript' );
+require( 'codemirror/mode/markdown/markdown' );
+require( 'codemirror/mode/pegjs/pegjs' );
+require( 'codemirror/mode/php/php' );
+require( 'codemirror/lib/codemirror.css' );
```

Comment:
there are more of these. if we had a way to async load them I would do so on-demand and provide the full suite with a better language selector

Reply:
Did you try using `import` instead of `require` here? It should teach Webpack to async load all those files. I guess we would still have to wrap `CodeMirror` into its own component to make sure those deps are loaded only when it mounts.

Reply:
I tried using `import()` and it didn't seem to recognize it. In d957054 I added the use of `require.ensure()` which seems to mostly work. There is still an issue that it's bundling the entirety of the modes directory into one chunk instead of many.